bin/mc-send-to-console

```diff
@@ -17,7 +17,7 @@ if [ ! -p "${CONSOLE_IN_NAMED_PIPE}" ]; then
   exit 1
 fi
 
-if [ "$(id -u)" = 0 ]; then
+if [ $UID != 0 ]; then
```

Comment:
Now understand what you were trying to solve. I think this is what will work for this scenario:

```suggestion
if [ "$(id -u)" = 0 -a $UID != 0 ]; then
```

This says "I'm starting out root (which is the default case) and I want to run the server as non-root user (the default case with UID=1000)".

Reply:
yeah, 
by default the server running as non-root user, and the command `mc-send-to-console` will execute by root, so its no problem.
but the pipe path `/tmp/minecraft-console-in` will be grant to root when the server running as root, so that if statement still to sudo execute an non-root user, and tooks error.